test/unit/theory/theory_int_opt_white.cpp

```diff
@@ -0,0 +1,137 @@
+#include <iostream>
+#include "api/cvc4cpp.h"
+#include "smt/optimization_solver.h"
+#include "expr/expr_manager.h"
+#include "expr/node.h"
+#include "expr/node_builder.h"
+#include "expr/node_manager.h"
+#include "expr/node_value.h"
+#include "smt/smt_engine.h"
+#include "smt/smt_engine_scope.h"
+#include "test_smt.h"
+
+namespace CVC4 {
+
+using namespace theory;
+using namespace expr;
+using namespace context;
+using namespace kind;
+using namespace smt;
+
+namespace test {
+
+class TestTheoryIntOptWhite : public TestSmt
```

Comment:
```suggestion
class TestTheoryWhiteIntOpt : public TestSmt
```

Reply:
The filename also needs to be renamed then.